.nvmrc

```diff
@@ -1 +1 @@
-v16.14.2
+v20.16.0
```

Comment:
Should the title of this PR be "Upgrade Node to v20.16.0", or should we be upgrading to v22.13.1 in the PR?

Reply:
yeah i started out pushing to upgrade to latest LTS but then belatedly realized that you can't actually specify `node22` as a github actions engine so went back to 20 and hadn't finished moving things back.

Reply:
Thanks for fixing it :)